builtin_string.go

```diff
@@ -952,58 +952,61 @@ func (r *Runtime) createStringIterProto(val *Object) objectImpl {
 	return o
 }
 
+func (r *Runtime) createString(val *Object) objectImpl {
+	o := r.newNativeFunc(r.builtin_String, r.builtin_newString, "String", r.global.StringPrototype, 1).self
```

Comment:
[q] What is the difference between getting the Object from `o = r.global.String.self` and what we're doing now? Is it because the Object doesn't fully exist because it is lazy loaded?

Reply:
> [q] Is there an added performance impact since we are creating a new Object here instead of how it was originally reassigning?

I havne't validated this but I think not because `newNativeFunc` creates and dereferences an object so we're just passing around a pointer. 

re-assigning r.global.String later on shouldn't do a copy of the value so i think there'd still be one allocation for this.